terraform/production/main.tf

```diff
@@ -76,3 +76,13 @@ resource "cloudfoundry_service_instance" "datafiles" {
   service_plan     = data.cloudfoundry_service.s3.service_plans["basic"]
   recursive_delete = true
 }
+
+data "cloudfoundry_service" "elasticsearch" {
+  name = "aws-elasticsearch"
+}
+
+resource "cloudfoundry_service_instance" "elasticsearch" {
+  name             = "es-prod"
+  space            = data.cloudfoundry_space.space.id
+  service_plan     = data.cloudfoundry_service.elasticsearch.service_plans["es-dev"]
```

Comment:
Should the plan be `.service_plans["es-medium"]`? I think we use medium in prod?